packages/relay/src/Environment.js

```diff
@@ -86,11 +89,25 @@ export default function createEnvironment(accessToken: string = '') {
               operation,
               variables,
             );
+
             if (store.check(operationSelector.root)) {
+              // This seems to work, but if we discover some problems with it
+              // Maybe we should rather return here
+              // I posted a question about it here: https://discordapp.com/channels/102860784329052160/102861057189490688
+              const root = {
+                ...operationSelector.root,
+                node: {
+                  ...operationSelector.root.node,
+                  selections: operationSelector.root.node.selections.filter(
+                    selection => selection.kind !== 'LinkedHandle',
+                  ),
+                },
+              };
+
               // we have all data in the store to fulfill this query so let's
               // load it from the memory first and call the API after that
               // this will make the UI feel really fast
-              observer.next(store.lookup(operationSelector.root));
+              observer.next(store.lookup(root));
```

Comment:
Could you please also catch the error here? Now it's hidden as you said.

Reply:
It is actually catched, now I finally understand were the weird error message on the all hotels list came from 🙈